app/code/AstoundDRudenko/PriceBadge/view/frontend/requirejs-config.js

```diff
@@ -0,0 +1,12 @@
+var config = {
+    config: {
+        mixins: {
+            'Magento_Catalog/js/price-box': {
+                'AstoundDRudenko_PriceBadge/js/product/mixin/hide-previous-price': true
+            },
+            'Magento_Swatches/js/swatch-renderer': {
+                'AstoundDRudenko_PriceBadge/js/product/mixin/price-badges': true
+            }
```

Comment:
preferably is to make a '-mixin' suffix to the file name instead of separate directory.
In case someone opens the file `app/code/AstoundDRudenko/PriceBadge/view/frontend/web/js/product/mixin/hide-previous-price.js` - he may treat it as a rewrite for some already existing mixing, but if the file name is - `app/code/AstoundDRudenko/PriceBadge/view/frontend/web/js/swatch-renderer-mixin.js` - there are no misunderstanding as it is clear that this is a mixin for a `swatch-renderer.js`.